test/connectFactoryObservable.test.tsx

```diff
@@ -20,9 +27,43 @@ describe("connectFactoryObservable", () => {
   })
   describe("hook", () => {
     it("returns the latest emitted value", async () => {
-      const [useNumber] = connectFactoryObservable((id: number) => of(id))
+      const valueStream = new BehaviorSubject(1)
+      const [useNumber] = connectFactoryObservable((id: number) =>
+        valueStream.pipe(map(v => `${id}:${v}`)),
+      )
       const { result } = renderHook(() => useNumber(1))
-      expect(result.current).toBe(1)
+      expect(result.current).toBe("1:1")
+
+      actHook(() => {
+        valueStream.next(3)
+      })
+      expect(result.current).toBe("1:3")
+    })
+
+    it("shares the subscription among all of the subscribers of the same key", async () => {
```

Comment:
Is the key not an implementation detail of `createFactoryObservable`?

Should this not be something like:

Multiple components using a hook returned from `createFactoryObservable`  with the same parameters share a multicasted subscription?

Reply:
Yep, that's a better test description 👍